apigee_edge.services.yml

```diff
@@ -48,6 +48,11 @@ services:
     tags:
       - { name: event_subscriber }
 
+  apigee_edge.breadcrumb.create_app:
+    class: Drupal\apigee_edge\Breadcrumb\CreateAppForDeveloperBreadcrumbBuilder
+    tags:
+    - { name: breadcrumb_builder, priority: 1001 }
```

Comment:
Based on a quick query on the core and open social 101 should be enough as a default priority, we do not need to be this drastic :D